src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.ShooterState;
+import frc.robot.subsystems.ArmSubsystem;
+import java.util.function.DoubleSupplier;
+
+/** An example command that uses an example subsystem. */
+public class ArmCommand extends Command {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final ArmSubsystem m_ArmSubsystem;
+
+  private final ShooterState m_shooterState;
+  private final DoubleSupplier m_yAxis;
+  private final double kMaxRotationsPerInput = 1000;
+
+  /**
+   * Creates a new ExampleCommand.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public ArmCommand(ArmSubsystem a_subsystem, ShooterState shooterState, DoubleSupplier yAxis) {
+    m_ArmSubsystem = a_subsystem;
+    m_shooterState = shooterState;
+    m_yAxis = yAxis;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(m_ArmSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if (m_yAxis.getAsDouble() != 0.0) {
+      m_ArmSubsystem.MoveArmRelative(m_yAxis.getAsDouble() * kMaxRotationsPerInput);
+
+    } else if (m_shooterState.isLoaded & !m_shooterState.isLowered) {
+      m_ArmSubsystem.lowerArm();
+      m_shooterState.lowered();
+    }
```

Comment:
Can you clarify this logic for me? Not sure I understand what you're trying to do. If the shooter is loaded and arm is not lowered, then lower the arm and set it to lowered state. I guess this is for detecting when we're loading at a human player station and we have loaded, but wouldn't this also interfere with the shooter operation when we're shooting? 

Possible solution, can we introduce a notion of `CycleState` similar to the `ShooterState`? That way we can better control what happens in each phase of the cycle. I'll have a longer think about how to best handle different cycle states as well, I think there is a broader change to state management here that's in order though maybe something for after the season

Reply:
> Can you clarify this logic for me? Not sure I understand what you're trying to do. If the shooter is loaded and arm is not lowered, then lower the arm and set it to lowered state. I guess this is for detecting when we're loading at a human player station and we have loaded, but wouldn't this also interfere with the shooter operation when we're shooting?
> 
> Possible solution, can we introduce a notion of `CycleState` similar to the `ShooterState`? That way we can better control what happens in each phase of the cycle. I'll have a longer think about how to best handle different cycle states as well, I think there is a broader change to state management here that's in order though maybe something for after the season

once it activates once it will save the state and not activate till fired and loaded again. 
this was so we could fold and fit under stage, can be removed but wanted to create global state for now

Reply:
Ok, will approve for now but let's remember this as a potential bug source.